### PR TITLE
quotas step

### PR DESCRIPTION
**Pull request by batzionb**

### CODE REVIEW

File: jest.config.ts [hunk 1]
```diff
@@ -25,10 +25,6 @@ const config: Config.InitialOptions = {
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
   testEnvironment: 'jsdom',
   setupFilesAfterEnv: ['<rootDir>/src/testUtils/setupTests.ts'],
-
-  transformIgnorePatterns: [
```
Comment (jtomasek) [inline jest.config.ts:29]: We should not compile whole node_modules. Let's figure out where the issue is coming from and compile just that dependency

**Review by jtomasek — CHANGES_REQUESTED** [review]

File: jest.config.ts (hunk 1 above)
Comment (batzionb) [reply jest.config.ts:29]: IIUC the default behavior of jest is to not compile node_modules
https://jestjs.io/docs/configuration#transformignorepatterns-arraystring
I copied this field from mce, it contains patterns that don't exist in our code and turns out this field is unnecessary altogether

**Review by jtomasek — APPROVED** [review]

**Pull request merged**